<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>game</class>
 <widget class="QDialog" name="game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../AdobeStock_364059453.jpeg</normaloff>../../../AdobeStock_364059453.jpeg</iconset>
  </property>
  <widget class="QLabel" name="CHESS">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1251</width>
     <height>571</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1 v 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>260</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>QUITTER</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>671</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-bottom-color: rgb(0, 0, 255);</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="button2_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>260</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IA 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonh">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>368</y>
     <width>83</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>HELP</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
